rasa/core/train.py

```diff
@@ -70,6 +72,36 @@ async def train(
     return agent
 
 
+async def train_in_chunks(
+    domain_file: Union[Domain, Text],
+    training_resource: "TrainingDataImporter",
```

Comment:
The import for it is not conditioned, so we can reference it without quotes.
```suggestion
    training_resource: TrainingDataImporter,
```